### ISSUE TITLE
[Feature Request] Save field selection per entry

### ISSUE THREAD

**Issue by joelzerner** [timkelty/craft-localesync#20]
#19 

The scenario is as I explained it originally in the mentioned issue, where the master content is all the entries in the main EN site, and the US site has mostly the same content, and there are other languages that have completely different content.

There are some instances where pages are different and shouldn't be synced. Currently, because the field selections aren't saved, it relies on the default settings of the field. In both cases, this leaves room for confusion: if the default is set to not sync, then every time an entry is saved, the user has to remember the turn the sync field on and make the selections there. If the default is set to sync automatically, then there's potential for the user to forget to turn sync off for the page(s) that don't need it and then content will get overwritten.

If the selected values for the Site Sync field were saved and remembered, then this issue could be avoided. The Site Sync field could be tucked away in a tab and only needed to be accessed to set up how an entry syncs in the first instance.

**Comment by timkelty** [comment]
Yep! I originally didn't include this, as it didn't fit our use-case, but it should be pretty easy.

I think it would have to be an option, per-field, though – as many times you'd _want_ it to revert to it's defaults as it does now.
